package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Stylus",
+  "version": "1.4.16",
+  "description": "Redesign the web with Stylus, a user styles manager",
+  "license": "GPL-3.0-only",
+  "repository": "openstyles/stylus",
+  "author": "Stylus Team",
+  "devDependencies": {
+    "archiver": "^2.1.1",
+    "codemirror": "^5.39.2",
+    "eslint": "^5.1.0",
+    "fs-extra": "^7.0.0",
+    "jsonlint": "^1.6.3",
+    "less": "^3.7.1",
+    "lz-string-unsafe": "^1.4.4-beta",
+    "semver-bundle": "^0.1.0",
+    "stylelint-bundle": "^8.0.0",
+    "stylus-lang-bundle": "^0.54.5",
+    "updates": "^4.0.0"
+  },
+  "scripts": {
+    "build": "npm run update-node && npm run update-versions && npm run build:cm",
+    "build:cm": "node tools/update-libraries.js && node tools/update-codemirror-themes.js",
```

Comment:
why `:`? what about `build-codemirror`?

Reply:
Shorter seemed better... I'll expand it.

Reply:
Well, it's not like we need to type it often, and if we do we can create a shortcut in our desktop/whatever or a batch/shell file.